src/OrchardCore.Modules/OrchardCore.ContentFields/Views/NumericField-Slider.Edit.cshtml

```diff
@@ -18,31 +20,58 @@
         <div class="col-md-6 col-lg-4">
             <label asp-for="Value">@name</label>
             <div class="input-group mb-2">
-                @*<div class="input-group-prepend">
-                    <div class="input-group-text">@min</div>
-                </div>*@
-                <input asp-for="Value" type="number" class="form-control content-preview-select" />
-                @*<div class="input-group-append">
-                    <div class="input-group-text">@max</div>
-                </div>*@
-            </div>
-            <div class="input-group mb-2 ml-2">            
-                @if (settings.Minimum.HasValue) {<span class="mr-3">@min</span>}
-                <input id="@(id)-slider" type="text" data-slider-min="@min" data-slider-max="@max" data-slider-step="@step" data-slider-value="@Model.Value" />            
-                @if (settings.Maximum.HasValue) {<span class="ml-1">@max</span>}
+                @if (settings.Minimum.HasValue)
+                {
+                    <div class="input-group-prepend">
+                        <div class="input-group-text">@min</div>
+                    </div>
+                }
+                <input asp-for="Value" class="form-control content-preview-select" placeholder="@settings.Placeholder" required="@settings.Required" />
+                @if (settings.Maximum.HasValue)
+                {
+                    <div class="input-group-append">
+                        <div class="input-group-text">@max</div>
+                    </div>
+                }
             </div>
         </div>
     </div>
+    <div class="input-group mb-2">            
+        @if (settings.Minimum.HasValue)
+        {
+            <span class="ml-1 mr-3">@min</span>
+        }
+        <input id="@(id)-slider" type="text" data-slider-min="@min" data-slider-max="@max" data-slider-step="@step" data-slider-value="@Model.Value" />            
+        @if (settings.Maximum.HasValue)
+        {
+            <span class="ml-3 mr-1">@max</span>
+        }
+    </div>
     @if (!String.IsNullOrEmpty(settings.Hint))
     {
         <span class="hint">@settings.Hint</span>
     }
 </div>
+
 <script at="Foot">
```

Comment:
asp-name ?

Reply:
Yeah ok it initializes the bootstrapSlider per field {id}

Reply:
Exactly, I need this script to render for each field.